content/docs/command-reference/plots/modify.md

```diff
@@ -30,6 +30,10 @@ Property names are passed as [options](#options) to this command (prefixed with
 `--`). These are based on the full
 [Vega specification](https://vega.github.io/vega/docs/specification/).
 
+Note that a secondary use of this command is to convert output or simple
+`dvc metrics` file into a plots file (see
```

Comment:
> `dvc metrics` file

I like this, we should probably apply in other places where manual links to "plots/metrics file(s)" exist. (In another PR).

Reply:
But don't worry about it @imhardikj, I'll take care of that.